### PR TITLE
Remove confusing items from DLC Table view

### LINKED ISSUE #2359: Contract maturity not working in the GUI
I am running on a0b60b2ebf954b4db185f09af95419b353794209

It appears we don't have contract maturity showing up. It would be nice if it was displayed in a nice format that makes it easy to determine when the contract matures

Screenshot from 2020-12-15 07-18-31

### PR DESCRIPTION
**Pull request by benthecarman**
Closes #2359

image

### CODE REVIEW

**Review by nkohen — COMMENTED** [review]
Should probably at least include the oracle event id no? Seems odd to have no oracle information at all

**Review by Christewart — APPROVED** [review]

**Comment by benthecarman** [comment]
> Should probably at least include the oracle event id no? Seems odd to have no oracle information at all

Added

**Comment by benthecarman** [comment]
Added a copy contract id button

File: app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCTableView.scala [hunk 1]
```diff
@@ -6,16 +6,21 @@ import scalafx.beans.property.StringProperty
...
+
+    val eventIdCol = new TableColumn[DLCStatus, String] {
+      text = "Event Id"
       prefWidth = 150
       cellValueFactory = { status =>
-        new StringProperty(status,
-                           "Temp Contract Id",
-                           status.value.tempContractId.hex)
+        val eventIdStr =
+          status.value.oracleInfo.singleOracleInfos.head.announcement.eventTLV.eventId
```
Comment (Christewart) [inline app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCTableView.scala:21]: is this always safe? I see `.head` which can throw no such element

**Review by Christewart — APPROVED** [review]

File: app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCTableView.scala (hunk 1 above)
Comment (benthecarman) [reply app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCTableView.scala:21]: Yeah, we have to have at least one oracle

**Pull request merged**